docs/runbook.md

```diff
@@ -41,15 +41,111 @@
 
 ### Configure Azure deployment
 
+Below section is for running cases on Azure platform, it specifies:
+  - admin_private_key_file: the private key file to access the Azure VM.
+  - subscription_id: Azure VM is created under this subscription.
+
+```yaml
+platform:
+  - type: azure
+    admin_private_key_file: $(admin_private_key_file)
+    azure:
+      subscription_id: $(subscription_id)
+```
+
 ### Select and set test cases
 
+Below section is to specify P0 and P1 test cases excluding case with name `hello`.
+
+```yaml
+testcase:
+  - criteria:
+      priority: [0, 1]
+  - criteria:
+      name: hello
+    select_action: exclude
+```
+
 ### Use variable and secrets
 
+Below section is to specify the variable in name/value format. We can use this variable in other field in this format `$(location)`.
```

Comment:
Group below content in bullet points. It can be clearer to see they are different. For example

* use inline variable
* use variable file
* mark a variable secret